proposals/0929-legacy-architecture-removal.md

```diff
@@ -0,0 +1,347 @@
+- [ ] `RCTBaseTextInputView`
+- [ ] `RCTBaseTextInputViewManager`
+- [ ] `RCTBaseTextShadowView`
+- [ ] `RCTBaseTextShadowViewEmbeddedShadowViewAttributeName`
+- [ ] `RCTBaseTextViewManager`
+- [ ] `RCTBridgeDelegate`
+- [ ] `RCTBridgeModuleListProvider`
+- [ ] `RCTConvert (UIActivityIndicatorView)`
+- [ ] `RCTConvert (UIScrollView)`
+- [ ] `RCTDebuggingOverlayManager`
+- [ ] `RCTImageShadowView`
+- [ ] `RCTImageView`
+- [ ] `RCTImageViewManager`
+- [ ] `RCTInputAccessoryShadowView`
+- [ ] `RCTInputAccessoryView`
+- [ ] `RCTInputAccessoryViewContent`
+- [ ] `RCTInputAccessoryViewManager`
+- [ ] `RCTJavaScriptCallback`
+- [ ] `RCTJavaScriptCompleteBlock`
+- [ ] `RCTJavaScriptExecutor`
+- [ ] `RCTModalHostView`
+- [ ] `RCTModalHostViewController`
+- [ ] `RCTModalHostViewInteractor`
+- [ ] `RCTModalHostViewManager`
+- [ ] `RCTModalManager`
+- [ ] `RCTModalViewInteractionBlock`
+- [ ] `RCTMultilineTextInputView`
+- [ ] `RCTMultilineTextInputViewManager`
+- [ ] `RCTObjcExecutorFactory`
+- [ ] `RCTProfileBeginAsyncEvent`
+- [ ] `RCTProfileCallbacks`
+- [ ] `RCTProfileDidEndProfiling`
+- [ ] `RCTProfileDidStartProfiling`
+- [ ] `RCTProfileEnd`
+- [ ] `RCTProfileEndAsyncEvent`
+- [ ] `RCTProfileGetQueue`
+- [ ] `RCTProfileHideControls`
+- [ ] `RCTProfileHookInstance`
+- [ ] `RCTProfileHookModules`
+- [ ] `RCTProfileImmediateEvent`
+- [ ] `RCTProfileInit`
+- [ ] `RCTProfileIsProfiling`
+- [ ] `RCTProfileRegisterCallbacks`
+- [ ] `RCTProfileSendResult`
+- [ ] `RCTProfileShowControls`
+- [ ] `RCTProfileTagAlways`
+- [ ] `RCTProfileUnhookModules`
+- [ ] `RCTRawTextShadowView`
+- [ ] `RCTRawTextViewManager`
+- [ ] `RCTRefreshControl`
+- [ ] `RCTRefreshControlManager`
+- [ ] `RCTRootContentView`
+- [ ] `RCTRootView`
+- [ ] `RCTRootView ()`
+- [ ] `RCTSafeAreaShadowView`
+- [ ] `RCTSafeAreaView`
+- [ ] `RCTSafeAreaViewLocalData`
+- [ ] `RCTSafeAreaViewManager`
+- [ ] `RCTScrollContentShadowView`
+- [ ] `RCTScrollContentView`
+- [ ] `RCTScrollContentViewManager`
+- [ ] `RCTScrollView`
+- [ ] `RCTScrollView (Internal)`
+- [ ] `RCTScrollViewManager`
+- [ ] `RCTSendFakeScrollEvent`
+- [ ] `RCTSinglelineTextInputView`
+- [ ] `RCTSinglelineTextInputViewManager`
+- [ ] `RCTSurface`
+- [ ] `RCTSurfaceRootShadowView`
+- [ ] `RCTSwitch`
+- [ ] `RCTSwitchManager`
+- [ ] `RCTTextAttributes`
+- [ ] `RCTTextAttributesIsHighlightedAttributeName`
+- [ ] `RCTTextAttributesTagAttributeName`
+- [ ] `RCTTextShadowView`
+- [ ] `RCTTextView`
+- [ ] `RCTTextViewManager`
+- [ ] `RCTVirtualTextShadowView`
+- [ ] `RCTVirtualTextView`
+- [ ] `RCTVirtualTextViewManager`
+- [ ] `RCTWrapperExampleView`
+- [ ] `RCTWrapperExampleViewController`
+- [ ] `RCTWrapperMeasureBlock`
+- [ ] `RCTWrapperReactRootViewController`
+- [ ] `RCTWrapperReactRootViewManager`
+- [ ] `RCTWrapperShadowView`
+- [ ] `RCTWrapperView`
+- [ ] `RCTWrapperView`
+- [ ] `RCTWrapperViewControllerHostingView`
+- [ ] `RCTWrapperViewManager`
+- [ ] `systrace_arg_t`
+- [ ] `UIScrollViewDelegate`
```

Comment:
I saw that [react-native-webview is using this](https://github.com/react-native-webview/react-native-webview/blob/5bc526fce5b9d6225df183bdf3d8cf542007d90a/apple/RNCWebViewImpl.m#L119), but I can't find in the deprecation notices if there is a replacement. Will we document the new alternatives?

Reply:
cc @cipolleschi